.docker/Dockerfile-newrelic

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:xenial-20210804 AS build
```

Comment:
I just wanted to put a reminder here that we'll need to add Ubuntu to our list of software components in the SSPP. 
And then I looked it up in "GEAR." Ubuntu version 18 LTS (Bionic) and 20 LTS (Focal) are approved. Approval for xenial expired. Can we use Focal?

Reply:
Or, alpine? I can't believe I didn't ask about that, although I can believe there's a good reason.

Reply:
I can use LTS